python/VPC_VPN_2_TUNNELS_UP/VPC_VPN_2_TUNNELS_UP.py

```diff
@@ -0,0 +1,384 @@
+# not use this file except in compliance with the License. A copy of the License is located at
+#
+#        http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+
+'''
+Description:
+  Checks that both the VPN tunnels provided by AWS Site-to-Site VPN are in UP status. The rule returns NON_COMPLIANT if one or both the tunnels are reporting status as DOWN.
+Trigger:
+  Configuration Changes
+Reports on:
+  AWS::EC2::VPNConnection
+Rule Parameters:
+  None
+Scenarios:
+  Scenario: 1
+     Given: At least one VPN tunnel provided by AWS Site-to-Site VPN connection has status as DOWN.
+      Then: Return NON_COMPLIANT with annnotation 'This AWS VPN Connection has at least one VPN tunnel down with the "statusMessage"(from configuration item)'.
+  Scenario: 2
+     Given: Both the tunnels provided by AWS Site-to-Site VPN Connection have status as UP.
+      Then: Return COMPLIANT
+'''
+
+import json
+import sys
+import datetime
+import boto3
+import botocore
+
+try:
+    import liblogging
+except ImportError:
+    pass
+
+##############
+# Parameters #
+##############
+
+# Define the default resource to report to Config Rules
+DEFAULT_RESOURCE_TYPE = 'AWS::EC2::VPNConnection'
+
+# Set to True to get the lambda to assume the Role attached on the Config Service (useful for cross-account).
+ASSUME_ROLE_MODE = False
+
+# Other parameters (no change needed)
+CONFIG_ROLE_TIMEOUT_SECONDS = 900
+
+#############
+# Main Code #
+#############
+
+def evaluate_compliance(event, configuration_item, valid_rule_parameters):
+    tunnel_1_status = configuration_item['configuration']['vgwTelemetry'][0]['status']
+    tunnel_2_status = configuration_item['configuration']['vgwTelemetry'][1]['status']
+
+    if tunnel_1_status == "DOWN":
+        return build_evaluation_from_config_item(
+            configuration_item,
+            'NON_COMPLIANT',
+            annotation='This AWS VPN Connection has at least one VPN tunnel down with statusMessage: ' + configuration_item['configuration']['vgwTelemetry'][0]['statusMessage']
+            )
+    if tunnel_2_status == "DOWN":
+        return build_evaluation_from_config_item(
+            configuration_item,
+            'NON_COMPLIANT',
+            annotation='This AWS VPN Connection has at least one VPN tunnel down with statusMessage: ' + configuration_item['configuration']['vgwTelemetry'][1]['statusMessage']
+            )
+    if tunnel_1_status == "UP" and tunnel_2_status == "UP":
+        return build_evaluation_from_config_item(configuration_item, 'COMPLIANT')
+    return None  # throwing pylint error Either all return statements in a function should return an expression, or none of them should without this

```

Comment:
The problem is that you are telling that there is only 2 value. What if something go wrong and another value get given?

Instead of None do
```
raise Exception("Unknown status of the VGW")
```